pm2.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    apps: [{
+        name: "pulpd",
+        script: "/opt/pulpd/site/main.js",
+        cwd: "/opt/pulpd/site/",
+        env_production: {
+            "NODE_ENV": "production"
+        },
```

Comment:
This is a super rough implementation of what this file might look like, but it basically controls the runtime environment of the event Pulp Fiction backend as managed by `pm2`. I can imagine we'd want to do some string injection as part of the deployment process, instead of using a hardcoded `/opt/pulpd/site/`, for example, or maybe add extra env vars.

But for now! Good enough.